DigitalLearningSolutions.Web/Controllers/LearningPortalController/Current.cs

```diff
@@ -165,5 +165,36 @@ public async Task<IActionResult> LaunchLearningResource(int learningLogItemId)
             // TODO: HEEDLS-678 redirect user to new LH forwarding endpoint.
             return Redirect(learningResourceLink);
         }
+
+        [HttpGet]
+        [Route("/LearningPortal/Current/MarkAsComplete/{learningLogItemId:int}")]

```

Comment:
Reminder to apply the ServiceFilter when 671 is merged.

Reply:
applied